tests/api/notify-callbacks.test.js

```diff
@@ -0,0 +1,189 @@
+import { createMocks } from "node-mocks-http";
+import notifyCallback from "../../pages/api/notify-callback";
+import { SQSClient } from "@aws-sdk/client-sqs";
+
+jest.mock("@aws-sdk/client-sqs");
+
+const sqsClient = {
+  send: jest.fn(),
+};
+
+describe("/api/notify-callbacks", () => {
+  beforeAll(() => {
+    process.env.GC_NOTIFY_CALLBACK_BEARER_TOKEN = "gc-form-super-secret-bearer-token";
+  });
+
+  afterAll(() => {
+    delete process.env.GC_NOTIFY_CALLBACK_BEARER_TOKEN;
+  });
+
+  it("Should reject GET request", async () => {
+    const { req, res } = createMocks({
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        Origin: "http://localhost:3000",
+      },
+    });
+
+    await notifyCallback(req, res);
+
+    expect(res.statusCode).toBe(403);
+    expect(JSON.parse(res._getData())).toEqual(expect.objectContaining({ error: "Forbidden" }));
+  });
+
+  it("Should reject request without authorization header", async () => {
+    const { req, res } = createMocks({
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Origin: "http://localhost:3000",
+      },
+    });
+
+    await notifyCallback(req, res);
+
+    expect(res.statusCode).toBe(403);
+    expect(JSON.parse(res._getData())).toEqual(
+      expect.objectContaining({ error: "Missing authorization header" })
+    );
+  });
+
+  it("Should reject request with wrong authorization header", async () => {
+    const { req, res } = createMocks({
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Origin: "http://localhost:3000",
+        authorization: "Bearer wrong_token_value",
```

Comment:
Yeah, it doesn't really matter that it's hard coded here in the test, but if you changed it, it would get rid of the Code Scanning error.